control/controlhttp/http_test.go

```diff
@@ -205,7 +205,7 @@ func testControlHTTP(t *testing.T, param httpTestParam) {
 		Logf:                 t.Logf,
 		omitCertErrorLogging: true,
 		testFallbackDelay:    50 * time.Millisecond,
-		Clock:                &tstest.Clock{},
```

Comment:
Worth noting that `tstest.Clock` supports fake "advances", which cause simulated time to move forward; you can create a clock with `NewClock` and the `Step` option:
https://pkg.go.dev/tailscale.com/tstest#ClockOpts

Reply:
I've spent some time trying to get this to work, but only got one of the tests passing with `tstest.Clock`. I'm not familiar with this package enough to get the advance hooks in all the right places.
OK if we merge this PR like this to unbreak CI for all other PRs, and update the other test later?